org.eclipse.jdt.ls.tests/src/org/eclipse/jdt/ls/core/internal/managers/GradleProjectImporterTest.java

```diff
@@ -106,7 +107,11 @@ public void testDisableGradleWrapper() throws Exception {
 			assertTrue(distribution instanceof WrapperGradleDistribution);
```

Comment:
assertTrue(WrapperGradleDistribution.class, distribution.class);

gives us more info in case of failed assertion